python/graphstorm/model/edge_gnn.py

```diff
@@ -22,9 +22,10 @@
 import dgl
 
 from .gnn import GSgnnModel, GSgnnModelBase
+from .gnn_encoder_base import fix_for_wholegraph
 from .utils import append_to_dict
 
-from ..utils import barrier, is_distributed, get_rank
+from ..utils import barrier, is_distributed, get_rank, use_wholegraph
```

Comment:
Calling use_wholegraph is inefficient as it is going to access local dir every time.
We can maintain a global variable instead.

Reply:
Done